lib/notification_report.rb

```diff
@@ -1,3 +1,5 @@
+require 'csv'
```

Comment:
Thanks, updated! As a side note, it looks like we don't run the lint checker this app... might pick that up at a later point.